src/main.scss

```
@@ -21,6 +21,8 @@
 // This targets the "content" node used by the Nextcloud wrapper.
 div#content {
   display: grid;
+  // To replace with `var(--color-main-background)` once dark mode support is
+  // implemented.
```

I suggest:
1. Making clear it's a todo (easier to catch in the editor with syntax highlighting)
2. Linking to a issue about support for dark theme in Cristal

```suggestion
  // TODO: replace with `var(--color-main-background)` once dark mode support is
  // implemented (see JIRAID-XXX)
```

---

Done